### DISCUSSION TITLE
Questions about the installation

### DISCUSSION THREAD

**Discussion by NotTheMindProbe** [LAGonauta/MetaAudio#70] [Q&A] [answered]
First, regarding manual installation - where's the "metahook" folder? I can't find any folder with that name in the repo, anywhere.
Second, regarding compilation and quick install - I tried compiling but the install bat for the title I want (HL Opposing Force) still says it can't find the compiled binaries. Plus running _build-initdeps.bat_ took just a few seconds and it only downloaded a single file

**Comment by lespaul64** [comment, marked as answer]
Let say for Steam Counter-Strike 1.6

```
Half-Life
	--> MetaHook.exe
	--> openal32.dll
	--> sndfile.dll
	--> phonon.dll
	
	--> cstrike
		--> metahook
			--> configs
				--> plugins.lst
			--> plugins
				--> MetaAudio.dll
```

Hope this will help you.

**Reply by NotTheMindProbe** [reply]
thank you!!
now, what about the compilation? I still don't understand what to do. Should I move the entire repo into the Half-Life installation and run all the BAT files there? or what?

**Reply by LAGonauta** [reply]
> thank you!! now, what about the compilation? I still don't understand what to do. Should I move the entire repo into the Half-Life installation and run all the BAT files there? or what?

No need to compile anything, you can just download latest master [here](https://github.com/LAGonauta/MetaAudio/releases/tag/master)
